2406495615_MadeShandyKrisnanda.py

```diff
@@ -1,112 +1,84 @@
+
+#Menyiapkan semua pengaturan dasar pada program ini
+t.title("Olympic Logo and Colorful Chessboard") 
 t.pensize(10)
 t.speed(0)
+t.screensize(500, 4500)
 t.colormode(255)
-pen_color = [0,129,200, 252,177,49, 0,0,0, 0,166,81, 238,51,78]
-def olympic_wave():
-    t.penup()
-    t.goto(-100,-50)
-    t.pendown()
-    for i in range(3):
-        t.pencolor((pen_color[i*6],pen_color[i*6+1],pen_color[i*6+2]))
-        t.circle(50)
-        t.penup()
-        t.forward(125)
-        t.pendown()
-    for i in range(2):
-        t.penup()
-        t.goto(-325/2,-90)
-        t.forward(125*(i+1))
-        t.pencolor((pen_color[i*6+3],pen_color[i*6+4],pen_color[i*6+5]))
-        t.pendown()
-        t.circle(50)
-        t.penup()
-        t.pendown()
-    for i in range(3):
-        t.penup()
-        t.pencolor((pen_color[i*6],pen_color[i*6+1],pen_color[i*6+2]))
-        t.home()
-        t.goto(-100, -50)
-        t.penup()
-        t.forward(125*i)
-        t.penup()
-        t.circle(50, 45)
-        t.pendown()
-        t.circle(50, 180)
-        t.penup()
-        t.circle(50, 90)
-        t.pendown()
-        t.circle(50, 45)
-        
-        
-def olympic_line():
-    t.speed(0)
+
+#Memasukan warna sesuai dengan kode RGB ke dalam variable dan list
+blue = 0,129,200
+yellow = 252,177,49
+black = 0,0,0
+red = 0,166,81
+green = 238,51,78
+pen_color = [blue, yellow, black, red, green]
+
+def make_circle(x, y, radius, degree, color): #Membuat lingkaran
+    t.pencolor(color)
     t.penup()
     t.home()
-    t.goto(-100,-220)
+    t.goto(x, y)
     t.pendown()
+    t.circle(radius, degree)
+
+def olympic_wave(): #Membuat logo olympic gelombang
+    for i in range(5): 
+        if i % 2 == 0 :
+            make_circle(-130+i*65,120,50,360,pen_color[i])
+        elif i % 2 == 1 :
+            make_circle(-130+i*65,80,50,360,pen_color[i])
     for i in range(5):
-        t.pencolor((pen_color[i*3],pen_color[i*3+1],pen_color[i*3+2]))
-        t.circle(50)
-        t.penup()
-        t.forward(125/2)
-        t.pendown()
-    t.penup()
-    t.goto(-100,-220)
-    t.speed()
-    for i in range(4):
-        t.pencolor(pen_color[i*3],pen_color[i*3+1],pen_color[i*3+2])
-        t.penup()
-        t.home()
-        t.goto(-100,-220)
-        t.forward(125*(i)/2)
-        t.circle(50, 90)
-        t.pendown()
-        t.circle(50, 90)
-        t.penup()
-        t.pendown()
-def asking_box():
+        if i % 2 == 0 :
+            make_circle(-130+i*65,120,50,-45,pen_color[i])
+            make_circle(-130+i*65,120,50, 70,pen_color[i])
+        elif i % 2 == 1 :
+            make_circle(-130+i*65,80,50,-135,pen_color[i])
+            make_circle(-130+i*65,80,50, 180,pen_color[i])
+
+def olympic_line(): #Membuat logo olympic lurus
+    for i in range(5):
+        make_circle(-130+i*65,-50,50,360,pen_color[i])
+    for i in range(5):
+        make_circle(-130+i*65,-50,50,-90,pen_color[i])
+        make_circle(-130+i*65,-50,50,180,pen_color[i])
```

Comment:
instead of defining "random number", define the variable from 1 point then calculate using specific formula